__docs__/wonder-blocks-button/button.argtypes.ts

```diff
@@ -7,8 +7,22 @@ export default {
         description: "Text to appear on the button.",
         type: {name: "string", required: true},
     },
-    icon: {
-        description: "A Phosphor icon asset (imported as a static SVG file).",
+    startIcon: {
+        description: `A Phosphor icon asset (imported as a static SVG file)
+            that will appear at the start of the button (left for LTR, right
+            for RTL).`,
```

Comment:
**praise:** Amazing as always, thanks for adding a11y docs even in the smallest details.